optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -438,7 +438,12 @@
         </listitem>
       </itemizedlist>
 
-      <para>You can also use a custom ranking, by implementing a <literal>Comparator</literal>:</para>
+      <para>All of the supported rankings do not rank solvers which have at least one failed single benchmark.</para>
+
+      <para>If <literal>Solver</literal>s have single benchmarks with uninitialized solutions,
+      they are by default ranked last - the bigger the sum of uninitialized variables in single benchmarks
+      for a given <literal>Solver</literal> is, the worse its ranking will be.
```

Comment:
"the bigger the sum of uninitialized variables... the worse its ranking " Is that still accurate in our new viewpoint? I might just state it more generally to avoid going into implementation details - I doubt anyone would care about them as the report raw data is clear for them to reverse engineer the ranking manually. In this particular case, I'd keep it short and focus on intent instead of specifics:
"Solvers with at least one failed benchmark don't get a ranking." [1] "Solvers with not fully initialized solutions are ranked worse."
[1] (because IIRC that's not even overridable - as it's a clear user error there is no need to make it overridable)